libqf/libqfcore/src/core/utils.cpp

```diff
@@ -81,37 +81,56 @@ bool Utils::fieldNameCmp(const QString &fld_name1, const QString &fld_name2)
 
 QVariant Utils::retypeVariant(const QVariant &val, int meta_type_id)
 {
-	if(meta_type_id == QVariant::Invalid) {
+	if(meta_type_id == QMetaType::UnknownType) {
 		//qfWarning() << "Cannot convert" << val << "to QVariant::Invalid type!";
 		// retype whatever to invalid variant
 		return QVariant();
 	}
 	if(val.userType() == meta_type_id)
 		return val;
+#if QT_VERSION >= QT_VERSION_CHECK(6, 4, 0)
```

Comment:
why not just `#if QT_VERSION_MAJOR >= 6`? Is there any specific feature in qt 6.4.0?

Reply:
No, it's just that I'm targeting 6.4.0 and I've copied this line from everywhere else. I can change it.

Reply:
Please, change it, to avoid questions like one in my previous post. :)

Reply:
the same here https://github.com/silicon-heaven/libshv/pull/233

Reply:
Yeah, I will change it everywhere

Reply:
Done